### DISCUSSION TITLE
Be a Muzei provider

### DISCUSSION THREAD

**Discussion by Cadair** [Radiokot/photoprism-android-client#18] [Ideas]
It would be amazing if this app could register a [Muzei](https://github.com/muzei/muzei) wallpaper provider so that Muzei could use the photos in a bookmarked search.

This is something I have dreamed of having the time to implement myself. If I ever try it I shall be sending you a PR :grin:

**Comment by Radiokot** [comment]
That's a good idea 💡 Will check out the API
